engines/decidim-debates/app/models/decidim/debates/debate.rb

```diff
@@ -23,6 +23,16 @@ def author_avatar_url
         author&.avatar&.url || ActionController::Base.helpers.asset_path("decidim/default-avatar.svg")
       end
 
+      # Public: Canpeople comment on this debate?
+      #
+      # Until we have a way to store options fore features and its resources we
+      # assume all debates can be commented.
+      #
+      # Returns Boolean
+      def commentable?
```

Comment:
Don't do it like this, use the settings like here: https://github.com/AjuntamentdeBarcelona/decidim/pull/638/files